README.md

```diff
@@ -64,60 +68,5 @@ upload = upload_file(
 layer_id = upload["layer_id"]
 ```
 
-### Uploading a Pandas DataFrame
```

Comment:
These interactions with well known libraries like Pandas and GeoPandas are probably some of the more attractive for Python folks so I think we should keep them here, just so they're featured prominently